README.md

```diff
@@ -182,6 +182,13 @@ There are a few important features and restrictions in this destination:
 
 This destination is using [Instabug iOS SDK](https://github.com/Instabug/Instabug-iOS) for logging implementation.
 
+### Analytics
+
+Analytics tracking is optional in logger framework, if client want to include analytics tracking they need to implement `AnalyticsProviderProtocol` and provide information through protocol and import `OktaLogger/Analytics` pod.
```

Comment:
@zeroes-ones can you add example code here, that indicates 
How to:
1. create a new Analytics instance 
2. How to add a analytics provide 
3. how to call track events on Analytics class

Also you had great sequence digram, can you edit it add it to the Doc ?